example.jl

```diff
@@ -0,0 +1,40 @@
+using Pkg; Pkg.activate("CMBLensingSPT3GInterface")
```

Comment:
Unfortunately, adding the PyCall `pytype_mapping` PR as a package dependency isn't enough to get auto-convert to work, I think since the PyCall pinned in the user's environment is the one that actually sets the auto-convert precedence in the wild. This is only a minor inconvenience, but it is a bit unsatisfying..

Reply:
Yea, there's no way to a package to specify a dependency on a non-tagged version. We can either 1) from the repo which _uses_ this package, pin the right PyCall version in that repo's Manifest or 2) include the code from my PyCall PR  in this package and just overrwrite the necessary function. I'd just go with 1 for now since pretty much we only ever use this from ultradeep100d or delensing, where we'd be activating that project's Manifest. But we could do 2 if somehow this get wider spread use and that PR doesnt get merged.

Reply:
Ah, so the problem is that the dependency is un-tagged, good to know. I would vote for 2), since I use this interface independently from any specific project pretty frequently. It's just copying over [these lines](https://github.com/JuliaPy/PyCall.jl/blob/b66bda1d10ac80dddd22c4d2abf47385269724f5/src/conversions.jl#L800-L821), right?

Reply:
But maybe there are other drawbacks to 2)?

Reply:
Only draw back will be a function redefinition warning, but I'l switch to going with option 2.

Reply:
(I was able to make it so that if you _do_ have my PR checked out, you wont get the function redefinition warning)

Reply:
Great!